cloud-database/src/db.json

```diff
@@ -0,0 +1,30 @@
+{
+    "Game-arooni" : {
+        "game3" : {
+            "playerCount" : 4,
+            "gameState" : "Softlocked by that god damn Smoke, Mungha Wurm, Damping Field, Static Orb combo",
+            "Players" : {
+                "player1" : {
+                    "life" : 40,
+                    "name" : "Scott",
+                    "Zones" : {

```

Comment:
FYI the schema in Firestore has changed. See #16

Reply:
I'll be updating this file later. This is just a placeholder for now.

(note the root element is Game-arooni ;) )